src/components/Avatar/Avatar.module.css

```diff
@@ -13,23 +13,35 @@
   justify-content: center;
 
   overflow: hidden;
-
-  color: var(--eds-theme-color-text-neutral-strong);
-  background-color: var(--eds-theme-color-background-neutral-medium);
-
   white-space: nowrap;
+
+  color: var(--eds-theme-color-text-utility-default-secondary);
+  background-color: var(--eds-theme-color-background-utility-default-low-emphasis);
+  border: calc(var(--eds-border-width-sm) * 1px) solid var(--eds-theme-color-border-utility-default-low-emphasis);
 }
 
-.avatar:focus-visible {
+/* use :has selector to put the focus ring on the container & apply the avatar's shape */
+*:has(> .avatar--is-interactive):focus-visible {
+  border-radius: calc(var(--eds-border-radius-full) * 1px);
+
   @mixin focus;
 }
 
 @supports not selector(:focus-visible) {
-  .avatar:focus {
+  *:has(> .avatar--is-interactive):focus {
+    border-radius: calc(var(--eds-border-radius-full) * 1px);
+
     @mixin focus;
   }
 }
 
+.avatar--is-interactive:hover,
+.avatar--is-interactive:active {
+  /* TODO-AH: use color token (Check with Design) */
```

Comment:
@janachoi For reference, didn't find the tier-2 token to use here, so have a hardcoded black value